sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/Microsoft.Azure.WebJobs.Extensions.EventGrid.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <Description>This extension adds bindings for EventGrid</Description>
+    <Version>3.0.0-beta.1</Version>
+    <NoWarn>$(NoWarn);AZC0001;CS1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../../../extensions/Microsoft.Azure.WebJobs.Extensions.Clients/src/Microsoft.Azure.WebJobs.Extensions.Clients.csproj" />
+    <ProjectReference Include="../../Azure.Messaging.EventGrid/src/Azure.Messaging.EventGrid.csproj" />
```

Comment:
Should this be a package reference?

Reply:
I would keep it as P2P while the extension is in development in case we would need to make library fixes to get it working.